scripts/generate-snippets.vsh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S v
```

Comment:
Thanks! Can you unwrap this from a shell script (so its a raw V file) and update the Makefile accordingly.

Reply:
Do you prefer raw "v" file or a v script file?

V script docs: https://github.com/vlang/v/blob/master/doc/docs.md#cross-platform-shell-scripts-in-v

Reply:
I've updated the `Makefile`